### DISCUSSION TITLE
Using Uniforms with instancing

### DISCUSSION THREAD

**Discussion by Amjad50** [bevyengine/bevy#10852] [Q&A]
Hello

I'm trying to build on the `instancing` example: https://github.com/bevyengine/bevy/blob/main/examples/shader/shader_instancing.rs

And I want to use uniforms, but I couldn't get it to work, I'm getting cryptic errors when I try to hack my way to make it work.
So I want to know if there is an intended way to do it?

Thanks

**Comment by mosure** [comment]
here is an implementation, a mashup of bevy mesh and shader_instancing:

https://github.com/mosure/bevy_gaussian_splatting/blob/70ef89579305bffd53db6c6354199510c3af2c4b/src/render/mod.rs#L533

**Reply by Amjad50** [reply]
Thanks a lot, let me try if it works for me
